README.md

```diff
@@ -0,0 +1,61 @@
+# Google Code-in Leaders
+
+***
+It is a website to show the current leaders from all organizations in GCI.
+***
+
+ * [Installation](https://github.com/learneradarsh/gci-leaders#installation)
+ * [Languages](https://github.com/learneradarsh/gci-leaders#languages)
+ * [Usage](https://github.com/learneradarsh/gci-leaders#usage)
+ * [Getting Involved](https://github.com/learneradarsh/gci-leaders#getting-involved)
+ * [Support](https://github.com/learneradarsh/gci-leaders#support)
+ * [Authors](https://github.com/learneradarsh/gci-leaders#authors)
+ * [License](https://github.com/learneradarsh/gci-leaders#license)
+
+***
```

Comment:
More of a stylistic choice, but I personally don't like this many breaks. It makes the file seem cluttered and don't add much.